src/css/search.scss

```diff
@@ -134,3 +134,11 @@ form.search-box {
     font-size: 36px;
   }
 }
+
+.listitem {
+  .topic-link {
+    float: left;
```

Comment:
Do you know what this is used for?

Reply:
`float:left` makes the topic move to the next line if it can't fit on the current one.